### ISSUE TITLE
expat-2.2.6-GCCcore-8.2.0.eb fails to configure on current centos7 as it wants docbook2X

### ISSUE THREAD

**Issue by jose-d** [easybuilders/easybuild-easyconfigs#7927]
`eb ./expat-2.2.6-GCCcore-8.2.0.eb -r` ([expat-2.2.6-GCCcore-8.2.0.eb](https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/e/expat/expat-2.2.6-GCCcore-8.2.0.eb)), #7853  fails in configure step with:
```
...
checking for working mmap... yes
checking for docbook2x-man... no
checking for db2x_docbook2man... no
checking for docbook2man... docbook2man
configure: error: Your local docbook2man was found to work with SGML rather
  than XML. Please install docbook2X and use variable DOCBOOK_TO_MAN to point
  configure to command docbook2x-man of docbook2X.
  Or use DOCBOOK_TO_MAN="xmlto man --skip-validation" if you have xmlto around.
  You can also configure using --without-docbook if you can do without a man
  page for xmlwf.
 (at easybuild/software/tools/EasyBuild/3.8.1/lib/python2.7/site-packages/easybuild_framework-3.8.1-py2.7.egg/easybuild/tools/run.py:501 in parse_cmd_output)
== 2019-03-19 16:07:42,612 easyblock.py:2870 WARNING build failed (first 300 chars): cmd " ./configure --prefix=/home/users/jose/.local/easybuild/software/tools/expat/2.2.6-GCCcore-8.2.0 --build=x86_64-pc-linux-gnu --host=x86_64-pc-linux-gnu " exited with exit code 1 and output:
checking for a BSD-compatible install... /usr/bin/install -c
checking whether build environment is sane..
== 2019-03-19 16:07:42,612 easyblock.py:288 INFO Closing log for application name expat version 2.2.6
```
os and relevant (IMO) rpms:
```
jose@koios1:~$ rpm -qa | grep docbook
docbook-utils-0.6.14-36.el7.noarch
docbook-style-xsl-1.78.1-3.el7.noarch
docbook-dtds-1.0-60.el7.noarch
docbook-style-dsssl-1.79-18.el7.noarch
jose@koios1:~$ cat /etc/centos-release
CentOS Linux release 7.6.1810 (Core) 
jose@koios1:~$
```

### workaround

`yum instal docbook2X`

This new dependency is required since expat `2.2.6`, expat `2.2.5` builds fine without `docbook2X`.

**Comment by jose-d** [comment]
to add `docbook2X` easyconfig should be trivial <sup>last famous words</sup>. WIP.

**Comment by jose-d** [comment]
ohoh. round dependency detected:
`docbook2X` needs `XML-Parser`
and
`XML-Parser` needs `expat`.

(and `expat` needs `docbook2X`)

so

> "You can also configure using --without-docbook if you can do without a man page for xmlwf."
perhaps this is the way.

**Comment by jose-d** [comment]
#7930 merged, with EasyBuild 3.9.0 is now problem no more reproducible, so closing this one.

```
jose@koios1:~/projects/ceico-easyconfigs$ eb /home/users/jose/.local/easybuild/software/tools/EasyBuild/3.9.0/lib/python2.7/site-packages/easybuild_easyconfigs-3.9.0-py2.7.egg/easybuild/easyconfigs/e/expat/expat-2.2.6-GCCcore-8.2.0.eb -r
== temporary log file in case of crash /tmp/eb-5kHQ1F/easybuild-Sb89TD.log
== resolving dependencies ...
== processing EasyBuild easyconfig /home/users/jose/.local/easybuild/software/tools/EasyBuild/3.9.0/lib/python2.7/site-packages/easybuild_easyconfigs-3.9.0-py2.7.egg/easybuild/easyconfigs/e/expat/expat-2.2.6-GCCcore-8.2.0.eb
== building and installing tools/expat/2.2.6-GCCcore-8.2.0...
== fetching files...
== creating build dir, resetting environment...
== starting iteration #0 ...
== unpacking...
== patching...
== preparing...
== configuring...
== building...
== testing...
== installing...
== taking care of extensions...
== restore after iterating...
== postprocessing...
== sanity checking...
== cleaning up...
== creating module...
== permissions...
== packaging...
== COMPLETED: Installation ended successfully
== Results of the build can be found in the log file(s) /home/users/jose/.local/easybuild/software/tools/expat/2.2.6-GCCcore-8.2.0/easybuild/easybuild-expat-2.2.6-20190514.114835.log
== Build succeeded for 1 out of 1
== Temporary log file(s) /tmp/eb-5kHQ1F/easybuild-Sb89TD.log* have been removed.
== Temporary directory /tmp/eb-5kHQ1F has been removed.
jose@koios1:~/projects/ceico-easyconfigs$ rpm -qa | grep -i docbook
docbook-utils-0.6.14-36.el7.noarch
docbook-style-xsl-1.78.1-3.el7.noarch
docbook-dtds-1.0-60.el7.noarch
docbook-style-dsssl-1.79-18.el7.noarch
jose@koios1:~/projects/ceico-easyconfigs$
```

**Comment by boegel** [comment]
Thanks a lot for following up on this @jose-d!
